### PR TITLE
People Management: Spinner

### LINKED ISSUE #5276: People Management: Spinner
#### Details:
Whenever `PeopleViewController` is onscreen, and there are no cached users, no spinner shows up onscreen.

We should indicate, somehow, that there's a network call going on.

**Notes:**
- PeopleViewController should always refresh the collection of users, as well.

### PR DESCRIPTION
**Pull request by jleandroperez**
Fixes #5276

To test:
1. Fresh install WPiOS
2. Log into a Dotcom account
3. Open `My Sites` > `Site` > `People`

As a result, a spinner should appear onscreen. On completion, it should get dismissed.
Plus: whenever the `People` screen gets displayed, the list of users will be refreshed. The spinner will only be visible when the list is empty.

Needs review: @diegoreymendez 
Thanks in advance Diego!

### CODE REVIEW

**Comment by jleandroperez** [comment]
@diegoreymendez sorry about the ping. Just found a UI glitch, please delay reviewing this.

Thank you!!

**Comment by jleandroperez** [comment]
Fixed!. @diegoreymendez ready for review.

Thank you!!

File: WordPress/Classes/ViewRelated/People/PeopleViewController.swift [hunk 1]
```diff
@@ -87,44 +86,60 @@ public class PeopleViewController: UITableViewController, NSFetchedResultsContro
...
         let person = Person(managedPerson: managedPerson)
         return person
     }
 
+    private func displayNoResultsIfNeeded() {
+        if resultsController.fetchedObjects?.count > 0 {
+            return
+        }
+
+        noResultsView.titleText = NSLocalizedString("Loading...", comment: "")
```
Comment (diegoreymendez) [inline WordPress/Classes/ViewRelated/People/PeopleViewController.swift]: Just wondering why the text is "Loading..." when we're showing there are no results.  Is this correct?

**Comment by diegoreymendez** [comment]
I'm not seeing the spinner... [video here](https://cloudup.com/cHR3ddn-_yQ).

**Comment by jleandroperez** [comment]
Apologies @diegoreymendez, i didn't update the issue description. I ended up wiring `WPNoResultsView` to indicate progress is being made (whenever there's nothing in the cache)

File: WordPress/Classes/ViewRelated/People/PeopleViewController.swift (hunk 1 above)
Comment (jleandroperez) [inline WordPress/Classes/ViewRelated/People/PeopleViewController.swift]: Yup, this is actually displayed when the cache is empty, and a backend call is taking place.

Thanks for checking!

**Comment by diegoreymendez** [comment]
:shipit:

**Comment by jleandroperez** [comment]
Thank you!!

**Pull request merged**